lib/rules/web/admin_console/4.8/logs.xyaml

```diff
@@ -106,5 +106,31 @@ toggle_log_wraps:
     selector:
       xpath: //input[@id='wrapLogLines']
     op: click
+check_logs_history_not_present:
+  elements:
+  - selector:
+      xpath: //button[@disabled]//span[text()='Current log']
+  - selector:
+      xpath: //div[@class='pf-c-select']
+    op: hover
+  - selector:
+      xpath: //div[@class='pf-c-tooltip__content' and contains(text(),'<message>')]
```

Comment:
use `contains` when matching element classes since it can easily change

Reply:
see combined comment for all class text matching comment below